x-pack/plugins/ml/public/application/explorer/explorer_utils.ts

```diff
@@ -600,3 +619,51 @@ export function removeFilterFromQueryString(
 
   return newQueryString;
 }
+
+// Returns an object mapping job ids to source indices which map to geo fields for that index
+export async function getSourceIndicesWithGeoFields(
+  selectedJobs: Array<CombinedJob | ExplorerJob>,
+  dataViewsService: any
```

Comment:
Please replace `any` with `DataViewsContract`
```suggestion
  dataViewsService: DataViewsContract
```